### ISSUE TITLE
Jupyter Notebook Previewer not working

### ISSUE THREAD

**Issue by chuabern** [conwnet/github1s#216]
![Uploading image.png…]()

when I click on `show preview` when trying to open up an .ipynb file, the preview is not working.

**Comment by xcv58** [comment]
Could you please share a reproducible link?

**Comment by chuabern** [comment]
@xcv58 i just tried it out and realized it's working fine with ipynb files in public repos, but when i open it from a private repo this error comes up

**Comment by conwnet** [comment]
> i just tried it out and realized it's working fine with ipynb files in public repos, but when i open it from a private repo this error comes up

@chuabern 

Yes. the jupyter extension use `jupyter.org` for a preview now, which cant access the private repos. And the special characters in the path/filename of the jupyter file also can be break the extension.

We will try to make a real jupyter viewer extension next.

**Comment by chuabern** [comment]
that sounds good! Looking forward.
